be/src/storage/lake/transactions.cpp

```diff
@@ -141,6 +141,8 @@ StatusOr<TabletMetadataPtr> publish_version(TabletManager* tablet_mgr, int64_t t
     // Save new metadata
     RETURN_IF_ERROR(log_applier->finish());
 
+    log_applier->print_apply_info();
```

Comment:
Why not just print it in finishe()?

Reply:
Because `finish` is also measured by this new add statistic log.

Reply:
this doesn't sound reasonable, measure `finish` should not prevent you from printing log.